jobs/deploy-znd.groovy

```diff
@@ -165,7 +165,10 @@ def determineVersion() {
    }
 
    def date = new Date().format("yyMMdd");
-   VERSION = "znd-${date}-${_currentUser()}-${params.VERSION}";
+   // VERSION parameter needs to be lowercased.
+   // Otherwise Fastly will have issue looking up the static version as it expects
+   // lowercase hostname.
+   VERSION = "znd-${date}-${_currentUser()}-${params.VERSION.toLowerCase()}";
```

Comment:
**suggestion (non-blocking):** maybe call `toLowerCase()` on the whole string afterwards.  I'm not sure if `_currentUser` is always lower case or not.